src/test/java/MainTest.java

```diff
@@ -9,4 +12,43 @@ public void testSth() {
         assertTrue(true);
     }
 
+
+    /**
+     * Method check setting path
+     * */
+    @Test
+    public void testFileManipulatorSetter() {
+        WordFinder fm = WordFinder.getInstance();
+
+        fm.setPath("data/temporary.txt");
+
+        assertTrue(fm.getPath().equals("data/temporary.txt"));
+    }
+
+
+    /**
+     * Method check sizeof List
+     * */
+    @Test
+    public void testListInWordFinder() {
+        WordFinder fm = WordFinder.getInstance();
+
+        fm.prepareKeywords();
+
+        assertTrue(fm.getSize() == 12);
```

Comment:
Tutaj ten test uzależniasz od jakiegoś pliku. Testy powinny być jak najbardziej niezależne od siebie i od innych czynników, takich jak właśnie pliki. I np. możesz zmockować wczytany plik na jakiś na sztywno. Jak chcesz, to pokażę Ci jak to się robi, to trochę trudniejsza sprawa, nigdy w Javie nie robiłem mocków ;)

Reply:
A, i tak samo - używaj assertEquals()